From 9c28f2130907c84b581f9f77dad62703225cc01e Mon Sep 17 00:00:00 2001
From: Stephane Danieau <stephane.danieau@st.com>
Date: Wed, 7 Aug 2024 09:05:22 +0200
Subject: [PATCH] arm64: defconfig: enable BT Low Energy feature

Add Bluetooth Low Energy (LE) feature

Change-Id: Idc2bac4b87ae6c65e3248b4601095e7f9eb566df
Signed-off-by: Stephane Danieau <stephane.danieau@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/397211
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Christophe ROULLIER <christophe.roullier@foss.st.com>
Reviewed-by: Stephane DANIEAU <stephane.danieau@foss.st.com>
Tested-by: Christophe ROULLIER <christophe.roullier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Christophe ROULLIER <christophe.roullier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 328b52b2e967..8a5595218b41 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,6 +1,7 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_GPIO=y
+CONFIG_BT_LE=y
 CONFIG_CLKSRC_STM32_LP=y
 CONFIG_CORESIGHT=y
 CONFIG_CORESIGHT_CPU_DEBUG=y
-- 
2.39.5

